include/cvc5/cvc5_proof_rule.h

```diff
@@ -1828,6 +1828,23 @@ enum ENUM(ProofRule)
    * \endverbatim
    */
   EVALUE(MACRO_ARITH_SCALE_SUM_UB),
+  /**
+   * \verbatim embed:rst:leading-asterisk
+   * **Arithmetic -- Non-linear multiply absolute value comparison**
+   *
+   * .. math::
+   *   \inferrule{F_1 \dots F_n \mid -}{F}
+   * 
+   * where :math:`F` is of the form 
+   * :math:`\left| t_1 \cdot t_n \right| \diamond \left| s_1 \cdot s_n \right|`.
+   * If :math:`\diamond` is :math:`=`, then each :math:`F_i` is
+   * :math:`\left| t_i \right| = \left| s_i \right|`. If :math:`\diamond` is :math:`>`, then
+   * each :math:`F_i` is either :math:`\left| t_i \right| > \left| s_i \right|` or
+   * :math:`\left| t_i \right| = \left| s_i \right| \land \left| t_i \right| \neq 0`.
```

Comment:
Mention that $F_1$ has the form $\left| t_1 \right| > \left| s_1 \right|$ if $\diamond$ is $>$.